web/src/App.css

```diff
@@ -26,6 +26,8 @@
   }
 }
 
-.card {
+.dualColumns {
```

Comment:
`logo-spin` is hyphen case but this is camel, should be one or the other, I vote hyphen case for css